src/app/pages/register/register.component.ts

```diff
@@ -129,6 +143,49 @@ export class RegisterComponent implements OnInit, OnDestroy {
     this.termsForm = this.formBuilder.nonNullable.group(termsControls);
   }
 
+  private applyFullNameLengthValidation(): void {
```

Comment:
I think you can still apply an Angular validator to the form to enforce this, even if it needs to work across two fields